modules/enabling_encapsulation.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-cpu-usage.adoc
+
+[id="enabling-encapsulation_{context}"]
+= Configuring mount namespace encapsulation
+
+You can configure mount namespace encapsulation so that the cluster runs with less resource overhead.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+
+* You have installed the OpenShift CLI (oc).
+
+.Procedure
+
+. Create a file called `mount_namespace_config.yaml` with the following YAML to control whether encapsulation is enabled or disabled:
+
++
+[source,yaml]
+----
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: master
+    name: 99-kubens-master
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+      systemd:
+        units:
+        - enabled: true <1>
+          name: kubens.service
+  ---
+  apiVersion: machineconfiguration.openshift.io/v1
+  kind: MachineConfig
+  metadata:
+    labels:
+      machineconfiguration.openshift.io/role: worker
+    name: 99-kubens-worker
+  spec:
+    config:
+      ignition:
+        version: 3.2.0
+      systemd:
+        units:
+        - enabled: true <2>
+          name: kubens.service
+----
+<1> If `enabled` is set to true, encapsulation is in effect for all control plane (previously known as "master") nodes.
+<2> If `enabled` is set to false, encapsulation is disabled for all worker nodes.
+
+. Apply the mount namespace `MachineConfig` CR:
++
+[source,terminal]
+----
+$ oc apply -f mount_namespace_config.yaml
+----
+
```

Comment:
Formatting is being thrown off. Needs `+` just before the .Example output line to tie it to previous. Apply to all steps that include code blocks and exampel outputs.
![image](https://user-images.githubusercontent.com/74046732/206197870-cc91a25b-5665-4de9-91a7-78ef78109fe4.png)

Reply:
Done.